roles/grafana_agent/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+# defaults file for agent
+
+# Mimir URL and creds
+agent_mimir_url: "http://sepia-grafana.front.sepia.ceph.com:9009/api/v1/push"
+agent_mimir_username: "admin"
+
+# Enable metrics globally
+agent_metrics: true
+
+# Labels used for external labels
+nodetype: "unknown_nodetype"
+
+scrape_interval_global: "60s"
```

Comment:
Why not put these values directly into `grafana-agent.yaml.j2`?

We should only template things that will change host-to-host, or things we expect other (non-Sepia) lab administrators to alter.

Reply:
This configuration is important because it determines the number of data points per minute (DPM) scraped in your active series. Depending on how you ship metrics to Grafana from the agent, you can reduce your total DPM by adjusting the scrape_interval setting, the same for the "scrape_interval_node" We can change those in the future in case overpopulated the Grafana server

Reply:
I Placed nodetype directly in grafana-agent.yaml.j2